tools/wpt/browser.py

```diff
@@ -1128,6 +1147,21 @@ def version(self, binary=None, webdriver_binary=None):
             self.logger.warning("Failed to find Edge binary.")
             return None
 
+    def webdriver_version(self, webdriver_binary):
+        if self.platform == "win":
+            return _get_fileversion(webdriver_binary, self.logger)
```

Comment:
Might want to check this on a Windows machine.

Reply:
Appears to work. Though amusing I found that `channel` isn't obeyed for choosing the Edge binary we grab, so if you have stable installed you always get stable 😆